x-pack/plugins/ml/public/application/datavisualizer/stats_table/components/field_data_expanded_row/keyword_content.tsx

```diff
@@ -5,21 +5,51 @@
  * 2.0.
  */
 
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import type { FieldDataRowProps } from '../../types/field_data_row';
 import { TopValues } from '../../../index_based/components/field_data_row/top_values';
+import { ChoroplethMap } from '../../../index_based/components/field_data_row/choropleth_map';
+import { useMlKibana } from '../../../../../application/contexts/kibana';
+import { EMSTermJoinConfig } from '../../../../../../../maps/public';
+import { COMMON_EMS_LAYER_IDS } from '../../../../../../common/constants/embeddable_map';
 import { DocumentStatsTable } from './document_stats';
 import { ExpandedRowContent } from './expanded_row_content';
+// import { EmbeddedMapComponent } from '../../embedded_map';
```

Comment:
Is this line needed?

Reply:
Removed in 614129ee1518c505f4b2686fa246e5967d58c361